bin/main.dart

```diff
@@ -6,8 +6,14 @@ import 'package:args/args.dart';
 void main(List<String> args) {
   final parser = ArgParser(allowTrailingOptions: true);
   parser.addMultiOption('localization', abbr: 'l', help: 'Localization codes');
+  parser.addOption('dirPath', abbr: 'd', defaultsTo: 'assets/i18n', help:'i18n files directory, \'assets/i18n\' by default');
```

Comment:
nice, 

btw, please set in AndroidStudio or IntelliJ in Settings(Preferences) -> Languages & Frameworks -> Flutter -> Editor - check Format code on save, check on Organize imports on safe.

This is need to automatically lint the Dart code and this can be cruicial becuase pub.dev runs the linting and complains that some part of the code is not "beautifill" and will reduce the pub points =)

When you will do this, and save your code, I bet this line will be divided by two for better readability